examples/data-objects/presence-tracker/tests/index.ts

```diff
@@ -0,0 +1,70 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Signaler } from "@fluid-experimental/data-objects";
+import {
+    IFluidContainer,
+    ContainerSchema,
+} from "fluid-framework";
+import {
+    TinyliciousClient,
+    TinyliciousContainerServices,
+} from "@fluidframework/tinylicious-client";
+import { FocusTracker, MouseTracker, renderFocusPresence, renderMousePresence } from "../src";
```

Comment:
I typically prefer to import directly from the location of each for tests (e.g. ../src/MouseTracker) because anything you export from index.ts is probably getting exported from the package as public API surface.  For this it doesn't matter much because we don't publish examples as npm packages, but it's still probably worth being careful about.